solana-programs/reward-manager/cli/src/main.rs

```diff
@@ -7,10 +7,22 @@ use clap::{
 
 use audius_reward_manager::{
     instruction::{
-        create_sender_public, create_sender, delete_sender, init, transfer, verify_transfer_signature,
+        create_sender_public,
+        create_sender,
+        delete_sender,
+        init,
+        transfer,
+        verify_transfer_signature,
+        delete_sender_public
     },
     processor::SENDER_SEED_PREFIX,
-    state::{RewardManager, SenderAccount, VerifiedMessages},
+    state::{
+        RewardManager,
+        SenderAccount,
+        VerifiedMessages,
+        DELETE_SENDER_MESSAGE_PREFIX,
+        ADD_SENDER_MESSAGE_PREFIX
```

Comment:
not sure i follow where these env vars are coming from (DELETE_SENDER and ADD_SENDER), but im sure this works if you've tested it 👍

Reply:
they're getting pulled in from the program state acct itself